.rubocop_todo.yml

```diff
@@ -9,20 +9,26 @@
 # Offense count: 1
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
-  Max: 42
+  Max: 28
 
 # Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
   Max: 227
 
+# Offense count: 8
+# Configuration parameters: Max.
+RSpec/ExampleLength:
+  Exclude:
+    - 'spec/features/availability_spec.rb'
+
 # Offense count: 3
 RSpec/ExpectActual:
   Exclude:
     - 'spec/routing/**/*'
     - 'spec/controllers/catalog_controller_spec.rb'
 
-# Offense count: 9
+# Offense count: 16
 # Configuration parameters: AggregateFailuresByDefault.
 RSpec/MultipleExpectations:
-  Max: 3
+  Max: 6
```

Comment:
I just added a exclusion on MultipleExpectations for all features. Cool with that?

Reply:
👍

Reply:
so how will we resolve this? I'll rebase my PR when this goes in to master and overwrite it? or if my PR goes in first we can rebase this branch? the race is on! ;)

Reply:
I rebased this branch 👍